docs/ref/django-admin.txt

```diff
@@ -1560,9 +1560,14 @@ useful if you need to create an initial superuser account or if you need to
 programmatically generate superuser accounts for your site(s).
 
 When run interactively, this command will prompt for a password for
-the new superuser account. When run non-interactively, no password
-will be set, and the superuser account will not be able to log in until
-a password has been manually set for it.
+the new superuser account.
+
+.. versionadded:: 3.0
+
+When run non-interactively, you can provide password by setting the
+`SUPERUSER_PASSWORD` environment variable otherwise, no password will be set,
+and the superuser account will not be able to log in until a password has been
+manually set for it.
```

Comment:
`versionadded` -> `versionchanged`, it should contain only changes, e.g.
```
When run interactively, this command will prompt for a password for
the new superuser account. When run non-interactively, you can provide password
by setting the ``SUPERUSER_PASSWORD`` environment variable otherwise, no
password will be set, and the superuser account will not be able to log in
until a password has been manually set for it.

.. versionchanged:: 3.0
    Support for using ``SUPERUSER_PASSWORD`` environment variable was added.                           
```